NIRSpec DR4 catalogue update

honzascholtz opened the pull request:
This is an update for the NIRSpec DR4 release for the morning of 1/10/2025 Europe time. I have added links and more info to the data.markdown with full info about DR4, while also adding a news item to the News section of the website.

honzascholtz committed:
first comit of DR4

honzascholtz committed:
update to the news

honzascholtz committed:
Update data.markdown

honzascholtz committed:
update to data access

honzascholtz committed:
fixed link

honzascholtz committed:
Fix link

honzascholtz committed:
Fixed link for the external table.

honzascholtz committed:
removing dummy links

honzascholtz committed:
Update data.markdown

honzascholtz committed:
minor edit.

brantr commented:
@honzascholtz Ryan is on vacation so I can help

Regarding the FitsMap -- the fitsmap links won't go active until the nircam DR5 release I assume? You mention that the spectra overlay are available but I do not think they are?

Otherwise this looks ready to merge, once the placeholder arxiv numbers and the herts website are known.

eclake committed:
Update data.markdown

Included link to MAST, mentioned timescale for fitsMap updates, and edited some other text.

eclake committed:
Update data.markdown

fixed typos and minor edits

eclake committed:
Update data.markdown

again more minor edits

eclake committed:
Update data.markdown

eclake committed:
Update data.markdown

eclake commented:
Hi @brantr - copying message from slack over to here 

I've made some updates that I can see on the pull request (minor).  Might you be able to merge with broken paper I and paper II links for now. I'll provide the arXiv links tomorrow but we're using this page as the main link in our papers to point to where people can find the data, so it's a bigger priority to get that implemented before the papers appear on arXiv
